build.gradle.kts

```diff
@@ -0,0 +1,71 @@
+import net.minecraftforge.gradle.user.tweakers.ClientTweaker
+import net.minecraftforge.gradle.user.tweakers.TweakerExtension
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+import org.spongepowered.asm.gradle.plugins.MixinGradlePlugin
+
+buildscript {
+    repositories {
+        maven { setUrl("http://files.minecraftforge.net/maven") }
+        maven { setUrl("http://repo.spongepowered.org/maven") }
+    }
+
+    dependencies {
+        classpath("net.minecraftforge.gradle:ForgeGradle:${Versions.forge_gradle}")
+        classpath("org.spongepowered:mixingradle:${Versions.mixin_gradle}")
+    }
+}
+
+group = "cc.hyperium"
+version = "1.0-SNAPSHOT"
+
+plugins {
+    kotlin("jvm") version Versions.kotlin
+}
+
+apply {
+    plugin<ClientTweaker>()
+    plugin<MixinGradlePlugin>()
```

Comment:
And these too?

Reply:
From my understand and experience, you cannot mix `plugins` and `apply` blocks, as they do two different tasks.